test/test_regression_cli.py

```diff
@@ -9,10 +9,13 @@
     sys.setdefaultencoding('latin-1')
 import pytest
 from retriever.lib.tools import getmd5
-from retriever import HOME_DIR, ENGINE_LIST
+from retriever import ENGINE_LIST
 
 mysql_engine, postgres_engine, sqlite_engine, msaccess_engine, csv_engine, download_engine, json_engine, xml_engine = ENGINE_LIST()
-dir_path = os.path.dirname(os.path.abspath(__file__))
+file_location = os.path.dirname(os.path.realpath(__file__))
+site_dir = os.path.abspath(os.path.join(file_location, os.pardir))
+working_script_dir = os.path.abspath(os.path.join(site_dir, "scripts"))
+HOMEDIR = os.path.expanduser('~')
```

Comment:
This concept was the main cause of all test failures. 
accessing the

 ```
SCRIPT_WRITE_PATH or DATA_SEARCH_PATHS
```
variables in tests, will not return the desired or intended path values. 
The issue is when we install retriever and run tests some of these paths if not used well will return paths relative to where the retriever was installed. 

Accessing `HOMEDIR` will always return a constant values. 

To set paths in test we should be careful and its better to start from
the location of the file `os.path.dirname(os.path.abspath(__file__))` and we find the retriever main directory.